Platform/CoffeelakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -111,10 +114,25 @@ UpdateFspConfig (
       PlatformId == PLATFORM_ID_CFL_S) {
     CopyMem (&Fspmcfg->SpdAddressTable, MemCfgData->SpdAddressTable, sizeof(MemCfgData->SpdAddressTable));
   } else {
-    Fspmcfg->MemorySpdPtr00       = (UINT32) MemCfgData->MemorySpdPtr00;
-    Fspmcfg->MemorySpdPtr01       = (UINT32) MemCfgData->MemorySpdPtr01;
-    Fspmcfg->MemorySpdPtr10       = (UINT32) MemCfgData->MemorySpdPtr10;
-    Fspmcfg->MemorySpdPtr11       = (UINT32) MemCfgData->MemorySpdPtr11;
+    MemSpd0CfgData = (MEM_SPD0_CFG_DATA *)FindConfigDataByTag (CDATA_MEM_SPD0_TAG);
```

Comment:
consider using a loop to simply the code and strings.

Reply:
Did not use loop but reduced code to couple of lines.